tests/aequilibrae/project/test_about.py

```diff
@@ -22,12 +22,20 @@ def tearDown(self) -> None:
         self.proj.close()
         rmtree(self.temp_proj_folder)
 
-    def test_create(self):
+    def test_create_and_list(self):
         self.proj.about.create()
 
         with self.assertWarns(expected_warning=Warning):
             self.proj.about.create()
 
+        list = self.proj.about.list_fields()
+        expected = ['model_name', 'region', 'description', 'author', 'license', 'scenario_name', 'year',
+                    'scenario_description', 'model_version', 'project_id', 'aequilibrae_version', 'projection']
+
+        failed = [x for x in list if x not in expected] + [x for x in expected if x not in list]
```

Comment:
cant you just use set difference there?

failed = Set(expected) - Set(list)

Reply:
I need a symetric difference, but it turns out that there is such a method

Reply:
xor right